restapi/src/main/java/io/stargate/web/resources/Db.java

```diff
@@ -45,6 +42,13 @@
           .maximumSize(10_000)
           .expireAfterWrite(Duration.ofMinutes(1))
           .build(this::getDocDataStoreForTokenInternal);
+
+  private final LoadingCache<TokenAndHeaders, AuthenticatedDB> restTokensToDataStore =
+      Caffeine.newBuilder()
+          .maximumSize(10_000)
```

Comment:
Could you elaborate more on why we pick 10_000 and expire after write 1 minute?

Reply:
Following the pattern @EricBorczuk had. Seemed like reasonable defaults.